RecoPPS/Local/test/re_alignment/reco_base_align_cfg.py

```diff
@@ -0,0 +1,71 @@
+import FWCore.ParameterSet.Config as cms
+
+from Configuration.StandardSequences.Eras import eras
+process = cms.Process('TEST', eras.Run2_2018)
+
+# minimum of logs
+process.MessageLogger = cms.Service("MessageLogger",
+  statistics = cms.untracked.vstring(),
+  destinations = cms.untracked.vstring("cout"),
+  cout = cms.untracked.PSet(
+    threshold = cms.untracked.string("WARNING")
+  )
+)
+
+# raw data source
+process.source = cms.Source("PoolSource",
+  #fileNames = cms.untracked.vstring(/store/data/Run2018D/ZeroBias/RAW/v1/000/320/688/00000/601A721D-AD95-E811-B21A-FA163E28A50A.root),
+  fileNames = cms.untracked.vstring("root://eoscms.cern.ch//eos/cms/store/group/phys_pps/sw_test_input/601A721D-AD95-E811-B21A-FA163E28A50A.root"),
```

Comment:
I suppose LFNs are more preferable
```suggestion
  fileNames = cms.untracked.vstring("/store/group/phys_pps/sw_test_input/601A721D-AD95-E811-B21A-FA163E28A50A.root"),
```

Reply:
Done